examples/step-86/step-86.cc

```diff
@@ -309,6 +309,17 @@ namespace Step86
   // derivative of the solution when computing the residual. We use the
   // `homogeneous_constraints` object for this purpose.
   //
+  // Note one detail here: The function
+  // AffineConstraints::make_consistent_in_parallel() might change the
```

Comment:
Perhaps it's worth replacing "change" by "expand"? The fact that the set may become larger (rather than smaller) is important in this context.